README.md

```diff
@@ -98,6 +98,15 @@ Hurray! You successfully made a contribution!
 
 ---
 
+## <ins> What if I have a Conflict? </ins>
+
+#### A GitHub conflict is when people make changes to the same area or line in a file. This must be fixed before it is merged in order to prevent collision in the main branch. 
```

Comment:
```suggestion
#### A GitHub conflict is when people make changes to the same area or line in a file. This must be fixed before it is merged to prevent collision in the main branch. 
```

Reply:
The first sentence would work as well. What do you think? I will resolve this you can discuss in the comment thread :)